src/flag_gems/fused/gelu_and_mul.py

```diff
@@ -11,7 +11,7 @@
 @triton.jit
 def gelu_none_and_mul_kernel(x, y):
     x_fp32 = x.to(tl.float32)
-    x_gelu = 0.5 * x_fp32 * (1 + tl.math.erf(x_fp32 * 0.7071067811))
+    x_gelu = 0.5 * x_fp32 * (1 + tl.extra.mlu.libdevice.erf(x_fp32 * 0.7071067811))
```

Comment:
What version of triton are you targeting now? It seems that this path to erf is in versions after 2.3.